buildlib/az-github-draft.yml

```diff
@@ -12,6 +12,13 @@ jobs:
     pool:
       name: MLNX
       demands: ${{ parameters.demands }}
+    variables:
+      ${{ if eq(variables['Build.Reason'], 'ResourceTrigger') }}:
+        POSTFIX: ucx-${{ replace(variables['Build.SourceBranch'], 'refs/heads/', '') }}
```

Comment:
why add ucx if you remove it in line 40 ?

Reply:
Removed, thanks